scripts/vulkaninfo_generator.py

```diff
@@ -567,15 +567,17 @@ def PrintStructure(struct, types_to_gen, structure_names, aliases):
                 out += f'        for (uint32_t i = 0; i < {v.arrayLength}; i++) {{ p.PrintElement(obj.{v.name}[i]); }}\n'
                 out += f"    }}\n"
             else:  # dynamic array length based on other member
-                out += f'    ArrayWrapper arr(p,"{v.name}", obj.' + v.arrayLength + ');\n'
-                out += f"    for (uint32_t i = 0; i < obj.{v.arrayLength}; i++) {{\n"
+                out += f"    {{\n"
+                out += f'        ArrayWrapper arr(p,"{v.name}", obj.' + v.arrayLength + ');\n'
+                out += f"        for (uint32_t i = 0; i < obj.{v.arrayLength}; i++) {{\n"
                 if v.typeID in types_to_gen:
-                    out += f"        if (obj.{v.name} != nullptr) {{\n"
-                    out += f"            p.SetElementIndex(i);\n"
-                    out += f'            Dump{v.typeID}(p, "{v.name}", obj.{v.name}[i]);\n'
-                    out += f"        }}\n"
+                    out += f"            if (obj.{v.name} != nullptr) {{\n"
+                    out += f"                p.SetElementIndex(i);\n"
+                    out += f'                Dump{v.typeID}(p, "{v.name}", obj.{v.name}[i]);\n'
+                    out += f"            }}\n"
```

Comment:
automatic code formatting? Not sure if this is your work or you didn't notice it happening. (Not holding up the PR, this is a nitpick)

Reply:
I added another set of braces around this code so indented everything inside. Here is what the output looks like: https://github.com/KhronosGroup/Vulkan-Tools/pull/833/files#diff-37d4883478e363cbe28b8d0750857d62686f218902043f53f263a51833d509dbR1972-R1980

Reply:
Ahhhhh 🤦 thats a bug on my part. ArrayWrapper will 'close' the output at the end of scope, which means braces need to be minded. Which they weren't, causing bad output formatting. Great catch!